sdk/servicebus/service-bus/test/perf/track-2/sbBase.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import { ServiceBusAdministrationClient, ServiceBusClient } from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = getEnvVar("SERVICEBUS_CONNECTION_STRING");
+const sbClient = new ServiceBusClient(connectionString);
+
+export abstract class ServiceBusTest<TOptions> extends PerfStressTest<TOptions> {
+  sbClient: ServiceBusClient;
+  sbAdminClient: ServiceBusAdministrationClient;
+  static queueName = `newqueue-${Math.ceil(Math.random() * 1000)}`;
+
+  constructor() {
+    super();
+    this.sbClient = sbClient;
+    this.sbAdminClient = new ServiceBusAdministrationClient(connectionString);
```

Comment:
Should `sbAdminClient` be static, since it's only used in global setup/cleanup?

Reply:
Yes, they get used only once, made them static.